contrib/completion/zsh/_docker

```diff
@@ -1481,6 +1481,7 @@ __docker_subcommand() {
         "($help)--log-driver=[Default driver for container logs]:logging driver:__docker_log_drivers"
         "($help)*--log-opt=[Log driver specific options]:log driver options:__docker_log_options"
         "($help)--mac-address=[Container MAC address]:MAC address: "
+        "($help)*--mount=[Attach a mount to the container]:mount: "
```

Comment:
done, it would be great if completions are automatically generated from the code..